.gitignore

```diff
@@ -127,7 +127,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
-
+/ffmpeg
 ffmpeg-2020-09-30-git-9d8f9b2e40-full_build/
```

Comment:
These two lines should be removed in favor of a wildcard filter `ffmpeg*` for more compatibility between multiple platforms

Reply:
```suggestion
ffmpeg*
```